src/gsi/obsmod.F90

```diff
@@ -769,7 +769,8 @@ subroutine init_obsmod_dflts
     if_model_dbz=.false.
     if_model_fed=.false.
     innov_use_model_fed=.false.
-    inflate_obserr=.false.
+!   increase error for reflectivity observation when |O-F| exceeds gross error magnitude
```

Comment:
By setting inflate_obserr = .true. in the code (not through GSI namelist), the inflation of obs error of dbZ is hardwired in GSI code. Is it possible to add "inflate_obserr" as a namelist option in gsimod (for example, in setup session or in radaruse_directDA session), so user could switch on/off the inflation in the run.  

If inflation of obs error for dbZ is a must-do for radar_dbZ DA in any senario, then it is OK by hardwire it in code.  Thank you!

Reply:
@GangZhao-NOAA Good idea.  I think we should add a namelist parameter.  I also propose renaming inflate_obserr to inflate_dbz_obserr.

Reply:
@daviddowellNOAA  Yes, renaming inflate_obserr to inflate_dbz_obserr is a good idea, since it is specifically applied for dbzDA.